dask_cudf/tests/test_core.py

```diff
@@ -115,6 +115,23 @@ def test_set_index(nelem):
         dd.assert_eq(expect, got, check_index=False, check_divisions=False)
 
 
+def test_timeseries_index():
+
+    df = pd.DataFrame()
+    df['date'] = pd.date_range('11/20/2018', periods=72, freq='D')
+    df['value'] = np.random.sample(len(df))
+
+    gdf = cudf.DataFrame.from_pandas(df)
+    ddf = dgd.from_cudf(gdf, npartitions=2)
+
+    ddf_ts_idx = ddf.set_index('date')
+
+    got = ddf_ts_idx.compute().to_pandas()
```

Comment:
```suggestion
    got = ddf_ts_idx
```

assert_eq will want to call `compute` for us.  This allows it to do things like verify that `ddf.dtypes == ddf.compute().dtypes` and other such checks.  It also calls `.to_pandas` I think if it's available.